src/constants.js

```diff
@@ -55,9 +55,9 @@ const MAX_NLEVELS = 48
 
 const WITHDRAWAL_CIRCUIT_NLEVELS = 32
 
-const WITHDRAWAL_WASM_URL = 'https://github.com/hermeznetwork/hermezjs/blob/main/withdraw-circuit-files/withdraw.wasm'
+const WITHDRAWAL_WASM_URL = 'http://raw.githubusercontent.com/hermeznetwork/hermezjs/main/withdraw-circuit-files/withdraw.wasm'
```

Comment:
Not https?

Reply:
Fixed. Somehow Github redirected to `http` when I clicked the links.